orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/model/GoogleCloudBuildStageDefinition.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.igor.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.api.services.cloudbuild.v1.model.Build;
+import lombok.Getter;
+
+@Getter
+public class GoogleCloudBuildStageDefinition {
+  private final String account;
+  private final Build buildDefinition;
+
+  // There does not seem to be a way to auto-generate a constructor using our current version of Lombok (1.16.20) that
```

Comment:
Jackson needs getters and setters for the class fields, so `@Data` on the class usually works here.

Reply:
Yeah, I really wanted an immutable class as the stage definition shouldn't be mutated (and won't actually be persisted if you do mutate it)...and this was the best way I could find to have Jackson deserialize into an immutable class.